lib/praxis/tasks/api_docs.rb

```diff
@@ -9,20 +9,21 @@
         raise Exception.new("NPM Install Failed")
       end
 
-      docs_dir=File.join(Dir.pwd, 'docs')
+      docs_dir = File.join(Dir.pwd, 'docs')
       FileUtils.mkdir_p docs_dir unless File.directory? docs_dir
 
       # The doc browser will need to have a minimal app.js and styles.css file at the root
-      # Let's add them if the app has not overriden them 
-      js_file = File.join(Dir.pwd, 'docs', 'app.js') 
-      scss_file = File.join(Dir.pwd, 'docs', 'styles.scss')   
+      # Let's add them if the app has not overriden them
+      js_file = File.join(Dir.pwd, 'docs', 'app.js')
+      scss_file = File.join(Dir.pwd, 'docs', 'styles.scss')
+      template_directory = File.expand_path(File.join(File.dirname(__FILE__), '../../../tasks/thor/templates/generator/empty_app/docs'))
+
       unless File.exists? js_file
-        File.open(js_file, 'w') {|f| f.write(%q{angular.module('DocBrowser', ['PraxisDocBrowser']);}) }
```

Comment:
Seems to me that you will want to have a single 'template' file rather than the same thing in two places. So for example you were missing some of the comments from those files. I expect that we may want to add a bit more to those files eventually, and it is better if you need to update only one place.